src/test/java/HomeWork20.java

```diff
@@ -0,0 +1,15 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork20 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        login("valentineguts@gmail.com", "te$t$tudent");
+        playlistClick("Valentino44");
+        platlistRemove();
+        Assert.assertTrue(getPlaylistRemovalMsg().isDisplayed());
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/224944784-18fb0450-57ad-4479-a163-c62564ab447e.png)

For example

![Test status](https://user-images.githubusercontent.com/104761482/224944889-69482a00-3b2a-46e8-ba33-094da100aaa2.png)

![Test error_](https://user-images.githubusercontent.com/104761482/224944958-7069b39a-04ce-4190-91a7-46c7da892237.png)

Please, using implicit wait just once in your code would be enough. It'l implicitly wait for every element that you try to find in your code.


` driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(4));`

![image](https://user-images.githubusercontent.com/104761482/224947458-805e86f8-b00f-4231-bffe-0b65cd3cd2bb.png)